test/integration/targets/vmware_inventory/runme.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+export ANSIBLE_CONFIG=ansible.cfg
+export vcenter_host="${vcenter_host:-0.0.0.0}"
+export vcenter_user="${vcenter_user:-user}"
+export vcenter_pass="${vcenter_pass:-pass}"
+VMWARE_CONFIG=test-config.vmware.yaml
+
+# flag for checking whether cleanup has already fired
+_is_clean=
+
+function _cleanup() {
+    [[ -n "$_is_clean" ]] && return  # don't double-clean
+    _is_clean=1
```

Comment:
The wrong lines were removed. The file cleanup is gone, but the _is_clean tracking still remains.

Reply:
OK.